### ISSUE TITLE
3.4 .sculpt_face_set is now supported, this should allow to import/export polygroups to sculpt mode now

### ISSUE THREAD

**Issue by kromar** [JoseConseco/GoB#368]
https://projects.blender.org/blender/blender/issues/51675#issuecomment-978400
https://wiki.blender.org/wiki/Reference/Release_Notes/3.4/Python_API#Internal_Mesh_Format
https://projects.blender.org/blender/blender/issues/51675

```
if ".sculpt_face_set" not in mesh.attributes:
    face_sets = mesh.attributes.new(".sculpt_face_set", "INT", "FACE")
    face_sets.data[10] = 14
```

**Comment by kromar** [comment]
https://wiki.blender.org/wiki/Reference/Release_Notes/3.4/Python_API#Internal_Mesh_Format

**Comment by fislysandi** [comment]
this would be such a poggers change doing polygrouping in zbrush then posing in blender with pose brush.

**Comment by kromar** [comment]
Face sets are available in this branch in version 4.1.5: https://github.com/kromar/blender_GoB/tree/extensions-platform-review

**Comment by kromar** [comment]
Thanks to @00004707 Face sets are available in this branch in version 4.1.5: https://github.com/kromar/blender_GoB/tree/extensions-platform-review
